src/components/render-page/word-counter.ts

```diff
@@ -0,0 +1,28 @@
+/*
+ * SPDX-FileCopyrightText: 2021 The HedgeDoc developers (see AUTHORS file)
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+
+import wordsCount from 'words-count'
+
+const EXCLUDED_TAGS = ['img', 'pre', 'nav']
+
+export const countWordsRecursive = (node: Element | ChildNode): number => {
+  let words = 0
+  if (
+    !node ||
```

Comment:
how can the node be false?

Reply:
Maybe a little `false` goes astray and is caught by this if?
Joke aside, changed it.